_ont_xgs/ont-fs-XGS-ONU-25-20NI.md

```diff
@@ -0,0 +1,586 @@
+| XGMII/XSGMII     | Yes                                                                        |
+| Optics           | SC/APC                                                                     |
+| IP address       | 192.168.100.1                                                              |
+| Web Gui          |                                                                            |
+| SSH              | No                                                                         |
+| Telnet           | ✅ user `PON S/N`, password `8digit HMAC-MD5 (PON S/N uppercase)`         |
+| Serial           | ✅                                                                        |
+| Serial baud      | 115200                                                                     |
+| Serial encoding  | 8-N-1                                                                      |
+| Form Factor      | miniONT SFP                                                                |
+
+{% include image.html file="XGS-ONU-25-20NI/front.jpg" alt="XGS-ONU-25-20NI front plate" caption="XGS-ONU-25-20NI front plate" %}
+{% include image.html file="XGS-ONU-25-20NI/internal.jpg" alt="XGS-ONU-25-20NI internal" caption="XGS-ONU-25-20NI internal layout" %}
+
+
+## Serial
+
+The stick has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be accessed from the SFP connector, but no components are mounted.
+
+The UART can be accessed by any of the following methods.
+- Touch the needle to a specific point
+- Shorting a specific pad to access from SFP
+
+### Access from PCB
+The UART can be accessed by connecting a wire or touching a needle to the following points.
+
+{% include image.html file="XGS-ONU-25-20NI/UART_needle.png" alt="XGS-ONU-25-20NI UART Touch point" caption="XGS-ONU-25-20NI UART Touch point" %}
+
+### Access from SFP
+By shorting these two points with solder, you can access the UART from SFP pins 2 and 7.
+
+| USB - TTL Adapter     | SFP Connector (Molex, etc) |
+| --------------------- | -------------------------- |
+| VDD (3.3V)            | pin #15 , #16              |
+| TX                    | pin #7                     |
+| RX                    | pin #2                     |
+| GND                   | pin #10                    |
+
+{% include alert.html content="USB TTL adapter may not work due to insufficient power supply. If possible, obtain 3.3V from the power supply device instead of the USB TTL adapter." alert="Note"  icon="svg-info" color="blue" %}
+
+{% include image.html file="XGS-ONU-25-20NI/UART_SFP.png" alt="XGS-ONU-25-20NI UART Short point" caption="XGS-ONU-25-20NI UART Short point" %}
+
+{% include image.html file="XGS-ONU-25-20NI/UART_bridge.png" alt="XGS-ONU-25-20NI UART solder bridge" caption="XGS-ONU-25-20NI UART solder bridge" %}
+
+
+## Firmware is interchangeable with
+The firmware is compatible with the following.
+- CIG XG-99S
```

Comment:
the page for CIG and EICN aliases should be created in this PR and should be redirected to this one for the time being